lib/rucio/daemons/conveyor/common.py

```diff
@@ -246,6 +246,9 @@ def bulk_group_transfer(transfers, policy='rule', group_bulk=200, fts_source_str
                 policy_key = '%s,%s' % (file['metadata']['src_rse'], file['metadata']['dst_rse'])
             if policy == 'rule_src_dest':
                 policy_key = '%s,%s,%s' % (transfer['rule_id'], file['metadata']['src_rse'], file['metadata']['dst_rse'])
+            if policy == 'activity_dest':
+                policy_key = '%s %s' % (activity, file['metadata']['dst_rse'])
```

Comment:
shouldn't there be a comma?

Reply:
No, the key can be whatever based on activity and dest